cv32/sim/uvmt_cv32/dsim.mk

```diff
@@ -354,5 +379,6 @@ clean:
 	rm -rf $(DSIM_RESULTS)
 
 # All generated files plus the clone of the RTL
-clean_all: clean clean_core_tests clean_riscv-dv clean_test_programs clean-bsp
+clean_all: clean clean_core_tests clean_riscv-dv clean_test_programs clean_compliance
 	rm -rf $(CV32E40P_PKG)
```

Comment:
Please restore the clean-bsp target as well as the clean_compliance

Reply:
Done!